modules/renter/contractor/consts.go

```diff
@@ -62,7 +67,11 @@ var (
 	maxCollateral    = types.SiacoinPrecision.Mul64(1e3) // 1k SC
 	maxDownloadPrice = maxStoragePrice.Mul64(3 * 4320)
 	maxStoragePrice  = types.SiacoinPrecision.Mul64(30e3).Div(modules.BlockBytesPerMonthTerabyte) // 30k SC / TB / Month
-	maxUploadPrice   = maxStoragePrice.Mul64(4320)
+	maxUploadPrice   = build.Select(build.Var{
+		Standard: maxStoragePrice.Mul64(4320),
+		Dev:      maxStoragePrice.Mul64(4320),
+		Testing:  maxStoragePrice.Mul64(4320e8),
```

Comment:
I feel like the better approach is to set a smaller allowance during testing, rather than allow a very high upload price to be set as part of our testing constants.